cos-fleetshard-support/src/main/java/org/bf2/cos/fleetshard/support/config/ApplicationOverrideConfigSourceProvider.java

```diff
@@ -0,0 +1,36 @@
+package org.bf2.cos.fleetshard.support.config;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.eclipse.microprofile.config.spi.ConfigSourceProvider;
+
+import io.smallrye.config.PropertiesConfigSource;
+
+public class ApplicationOverrideConfigSourceProvider implements ConfigSourceProvider {
+    private static final String OVERRIDE_PROPERTIES_LOCATION = "OVERRIDE_PROPERTIES_LOCATION";
+
+    @Override
+    public Iterable<ConfigSource> getConfigSources(ClassLoader forClassLoader) {
+        String overridePropertiesLocation = System.getenv(OVERRIDE_PROPERTIES_LOCATION);
+        if (overridePropertiesLocation == null) {
+            return Collections.EMPTY_LIST;
+        }
+        File f = new File(overridePropertiesLocation);
+        if (f.exists() && !f.isDirectory()) {
```

Comment:
If the File doesn't exist I'd add a Log.warn to let someone know the env var points to an invalid file.

Reply:
done